src/upstream_rbus.c

```diff
@@ -53,6 +61,25 @@ void subscribeRBUSevent()
         ParodusInfo("rbusEvent_Subscribe was successful\n");
 }
 
+#ifdef WAN_FAILOVER_SUPPORTED
+/* API to subscribe Active Interface name on value change event*/
+int subscribeCurrentActiveInterfaceEvent()
+{
+	int rc = RBUS_ERROR_SUCCESS;
+	ParodusInfo("Subscribing to Device.X_RDK_WanManager.CurrentActiveInterfaceEvent..\n");
+	rc = rbusEvent_Subscribe(rbus_Handle,WEBPA_INTERFACE,eventReceiveHandler,"parodusInterface",20);
+        if(rc != RBUS_ERROR_SUCCESS)
+	{
+		ParodusInfo("Device.X_RDK_WanManager.CurrentActiveInterface subscribe failed : %d\n", rc);
```

Comment:
Move this as ParodusError , and use WEBPA_INTERFACE macro in log instead of direct TR181 name in log